src/App.tsx

```diff
@@ -1,26 +1,98 @@
 import React from 'react';
-import logo from './logo.svg';
-import './App.css';
+import { useState } from "react";
+import { makeStyles } from "@mui/styles";
+import { Button } from "@mui/material";
+import { createTheme, ThemeProvider, styled } from '@mui/material/styles';
+import Container from '@mui/material/Container';
+import Typography from '@mui/material/Typography';
+import { Box } from "@mui/system";
+
+
+declare module '@mui/material/styles' {
+  interface Theme {
+    status: {
+      secondary: string;
+    };
+  }
+  interface ThemeOptions {
+    status?: {
+      secondary?: string;
+    };
+  }
+}
+
+const theme = createTheme({
+  status: {
+    secondary: 'burlywood',
+  },
+});
```

Comment:
Let's move this into it's own file and give it a more descriptive name, like challengeTheme or something like that.

Reply:
This change is for accomplish a cleaner code. Every file should have a clear purpose and a single responsibility as should every component, function, class etc.